src/main/java/com/ss/ucm/ms/loans/services/LoanAdd.java

```diff
@@ -20,54 +21,92 @@
 @Service
 @Transactional
 public class LoanAdd {
-	
+
 	@Autowired
 	private LoanSearch loanSearch;
-	
+
 	@Autowired
-    private LoanDAO loanDAO;
-	
-    
-    @Autowired
-    UserLoanAdd userLoanAdd;
-    
+	private LoanDAO loanDAO;
+
+
+	@Autowired
+	UserLoanAdd userLoanAdd;
+
 	/**
 	 * 
 	 * @param loan
 	 * @return Loan(Entity)
 	 * 
 	 */
-	
-    public Loan saveLoan(Loan loan) {
-    	return loanDAO.save(loan);
-    }
-    
-    
-      
+
+	public Loan saveLoan(Loan loan) {
+		return loanDAO.save(loan);
+	}
+
+
+
 	/**
 	 * 
 	 * @param userId, loanRequest(body)
 	 * @return void
 	 * 
 	 */
-    public void signUpLoan(int userId, RequestLoanSignupDto loanRequest) {
-    	Loan loan = new Loan();
-		
-		loan.setMaxAmount(loanRequest.getMaxAmount());
-		loan.setName(loanRequest.getName());
-		loan.setInterestRate(loanRequest.getInterestRate());
-				
-		saveLoan(loan);
+	public void signUpLoan(int userId, RequestLoanSignupDto loanRequest) {
 
 		UserLoan userLoan = new UserLoan();
 		userLoan.setLoanId(loanSearch.LoanId(loanRequest.getName()));
 		userLoan.setUserId(userId);
+		userLoan.setSalary(loanRequest.getSalary());
 		userLoan.setBalance(loanRequest.getBalance());
 		userLoan.setDate(loanRequest.getStartDate().now());
```

Comment:
This will always set the start date to the current time no matter what loanRequest says, since `LocalDate.now()` is a static method and doesn't care about the instance's data.